theme/static/js/hs_file_browser.js

```diff
@@ -1611,6 +1611,45 @@ $(document).ready(function () {
         return false;
     });
 
+    // User clicked Proceed button on invalid URL warning dialog - need to add url without validation
+    $("#btn-reference-url-without-validation").click(function () {
+        var resID = $("#hs-file-browser").attr("data-res-id");
+        var currentPath = $("#hs-file-browser").attr("data-current-path");
+        var refName = $("#ref_name_passover").val();
+        var refURL = $("#ref_url_passover").val();
+        var newRefURL = $("#new_ref_url_passover").val();
+        var calls = [];
+        if (refName && refURL) {
+            calls.push(add_ref_content_ajax_submit(resID, currentPath, refName, refURL, false));
+
+            // Disable the Cancel button until request has finished
+            $(this).parent().find(".btn[data-dismiss='modal']").addClass("disabled");
+
+            function afterRequest() {
+                refreshFileBrowser();
+                $("#btn-reference-url-without-validation").parent().find(".btn[data-dismiss='modal']").removeClass("disabled");
+            }
+
+            $.when.apply($, calls).done(afterRequest);
+            $.when.apply($, calls).fail(afterRequest);
```

Comment:
You could us always to replace done and fail.  https://api.jquery.com/deferred.always/

Reply:
@sblack-usu Thanks for the pointer. I'll leave this to @Maurier to do this replacement as needed on a separate PR since the same ```done``` and ```fail``` callback logic are used throughout this file, and I just followed suit here.